gvsbuild/projects/libadwaita.py

```diff
@@ -25,9 +25,9 @@ def __init__(self):
             self,
             "libadwaita",
             repository="https://gitlab.gnome.org/GNOME/libadwaita",
-            version="1.5.3",
```

Comment:
I saw you opened tickets in upstream libadwaita... is this working or we need to resolve those issues?

Reply:
Hi @pbor, it works as long as tests are turned off.